tests/testthat/test-RandMat.R

```diff
@@ -0,0 +1,239 @@
+context("RandMat* tests")
+library(rerf)
+
+test_that("Testing RandMat binary option.", {
+  ## below is the output of RandMat with mat.options
+  ## from commit 73b896ff053537ee23d82b9debee054171b1c41b
+  ## with set.seed(317) and RcppZiggurat::zsetseed(14)
+  ## for comparison to the new version of RandMat*
+  #mat.options <- list(p = 5, d = 3, "binary", rho = 0.25, prob = 0.5)
+  rBinary <- structure(c(3, 2, 3, 2, 1, 2, 2, 3, 1, -1, -1, 1), .Dim = 4:3)
+
+  set.seed(317)
+  RcppZiggurat::zsetseed(14)
+  #optBinary <- list(p = 5, d = 3, "binary", rho = 0.25, prob = 0.5)
+  #a01 <- RandMat(optBinary)
+  optBinary <- list(p = 5, d = 3, rho = 0.25, prob = 0.5)
```

Comment:
`rho` is not a parameter for randmatbinary (get a warning when I try to run testthat)